### PR TITLE
Rebuild for aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by xhochy**
Skips `ipykernel` related tests to break the dependency cycle.

### CODE REVIEW

**Comment by xhochy** [comment]
`trio` is missing on the mirror currently, so we need to retrigger the builds once the CDN is synced again.

**Comment by Carreau** [comment]
We can also remove the trio dependency for tests; It is not necessary for IPython to work properly; and it is necessary only to test trio integration; same for curio.

**Comment by xhochy** [comment]
> We can also remove the trio dependency for tests; It is not necessary for IPython to work properly; and it is necessary only to test trio integration; same for curio.

No, as this is only an infrastructure hiccup it will work in some time. Otherwise, we still need to wait for `matplotlib` to be rebuild for `aarch64`. It depends on `gtk2` and `qt2` which are still in the queue for rebuilds.

**Comment by hmaarrfk** [comment]
Sorry should we skip the tri and matplotlib tests for now? it seeks that this package is being here
```
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 244, in verify_specs
    raise ResolvePackageNotFound(bad_deps)
conda.exceptions.ResolvePackageNotFound: 
  - matplotlib
  - trio
```

I'm sure the graphical stacks will have different challenges, and it might be good to be able to build up the stacks independently.

**Comment by xhochy** [comment]
@conda-forge/ipython This is ready for review/merge.

**Comment by bollwyvl** [comment]
LGTM, sorry for the delay in review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,15 +39,15 @@ requirements:
...
     - nbformat
     - nose>=0.10.1
     - numpy
     - pygments
     - requests
     - testpath
-    - trio
-    - matplotlib
+    - trio   # [not (aarch64 or ppc64le)]
+    - matplotlib  # [not (aarch64 or ppc64le)]
```
Comment (bollwyvl) [inline recipe/meta.yaml]: :+1:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -71,7 +71,7 @@ test:
   commands:
     - ipython -h
     - ipython3 -h
-    - NOSE_EXCLUDE="test_history|recursion" iptest  # [unix]
+    - NOSE_EXCLUDE="test_history|recursion" iptest  # [unix and not (aarch64 or ppc64le)]
```
Comment (bollwyvl) [inline recipe/meta.yaml]: is the intention to restore tests at some point?

**Pull request merged**